client/modules/Lane/LaneActions.js

```diff
@@ -1,41 +1,91 @@
-import uuid from 'uuid';
+//import uuid from 'uuid';
```

Comment:
jakiś powód zostawiania wykomentowanego kodu? Jeśli to takie "zostawianie na później, bo może się przyda" to jest to no go :P